scripts/security/clean-invisible-chars.py

```
@@ -0,0 +1,113 @@
+    results = {}
+    total_changed = 0
+    for base in (os.path.join(root, "src"), os.path.join(root, "docs")):
+        if not os.path.isdir(base):
+            continue
+        for dirpath, _, filenames in os.walk(base):
+            for fname in filenames:
+                ext = os.path.splitext(fname)[1].lower()
+                if ext not in EXTENSIONS:
+                    continue
+                full = os.path.join(dirpath, fname)
+                cleaned, delta = clean_file(full) if not dry_run else (False, 0)
```

Ruff (0.15.9)

[warning] Unpacked variable `cleaned` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)